etl/pad/raw/skills/behaviors/active_behaviors.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Dict, List, Literal, NamedTuple, Optional, TYPE_CHECKING, Union
+
+from pad.raw.skills.behaviors.buffs import Buff, Debuff
+
+if TYPE_CHECKING:
+    from pad.raw.skills.active_skill_info import ActiveSkill
+
+
+class ASBOrbChange(NamedTuple):
+    from_orbs: List[int]
+    to_orbs: List[int]
+    amount: int = 999
+    from_invert: bool = False
+
+    behavior_type = 'orb_change'
+
+
+class ASBBoardChange(NamedTuple):
+    board_shape: List[List[int]]
+
+    behavior_type = 'board_change'
+
+
+class ASBDamage(NamedTuple):
+    unit: Literal['flat',
+                  'multiplier', 'team_multiplier', 'hp_multiplier', 'grudge_multiplier',
+                  'gravity', 'true_gravity']
+    amount: Union[int, float]
+    min_damage: Optional[Union[int, float]] = None
+    attribute: Optional[int] = None
+    recover: float = 0.0
+
+    mass_attack: bool = True
+    laser: bool = False
+    team_mult_attr: Optional[List[int]] = None
+
+    target_attribute: Optional[int] = None
+
+    behavior_type = 'damage'
+
+
+class ASBRecover(NamedTuple):
+    unit: str
+    amount: Union[int, float]
+
+    skill_bind: int = 0
+    awoken_bind: int = 0
+    match_bind: int = 0
+
+    behavior_type = 'recover'
+
+
+class ASBBuff(NamedTuple):
+    buff_type: Literal['shield', 'stat_mult', 'counterattack', 'mass_attack', 'skyfall_chance',
+                       'orb_movement', 'attribute_change', 'enemy_attribute_change', 'increase_combo',
+                       'void', 'increase_skyfall', 'disable_skills', 'unmachable', 'autoheal',
+                       'no_skyfall', 'spinner']
+    turns: int
+    details: Buff = {}
+
+    behavior_type = 'buff'
+
+
+class ASBInflictDebuff(NamedTuple):
+    debuff_type: Literal['delay', 'guard_break', 'poison']
+    details: Debuff = {}
+
+    behavior_type = 'inflict_debuff'
+
+
+class ASBSuperSkill:
```

Comment:
maybe name this `ASBMultipart` to be in line with `ASMultipart`? since it's the special case "vector" container of skill behaviors